src/program/mod.rs

```diff
@@ -44,8 +46,21 @@ impl<'a> Program<'a> {
         texture_creator: &'a TextureCreator<WindowContext>,
         args: cli::Args,
     ) -> Result<Program<'a>, String> {
-        let images = args.files;
+        let args = cli::cli()?;
```

Comment:
Could you remove this line:
`         let args = cli::cli()?;`

cli argument parsing is done in `src/main.rs`.